src/features/upload-file/build-path.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path'
+import { BaseRecord } from 'admin-bro'
+
+/**
+ * Creates a path to the file. Related to the given provider. If it is an AWS
+ * path is related to the bucket.
+ *
+ * @param   {BaseRecord}  record
+ * @param   {string}      path        file path
+ *
+ * @return  {string}
+ * @private
+ */
+const buildRemotePath = (
+  record: BaseRecord,
+  filePath: string,
+): string => {
+  if (!record.id()) {
+    throw new Error('you cannot upload file for not persisted record. Save record first')
+  }
+  const { ext, name } = path.parse(filePath)
```

Comment:
Previously it was set to the value stored in "filename" property. I changed it that it stores files under original filename and then (later in the code) it stores this filename when `properties.filename` is set.

So we wont have problems with case where user didn't fill filename property in the UI